src/h3lib/include/h3api.h.in

```diff
@@ -319,14 +319,14 @@ DECLSPEC double H3_EXPORT(radsToDegs)(double radians);
  * @{
  */
 /** @brief "great circle distance" between pairs of LatLng points in radians*/
-DECLSPEC double H3_EXPORT(distanceRads)(const LatLng *a, const LatLng *b);
+DECLSPEC double H3_EXPORT(latLngDistanceRads)(const LatLng *a, const LatLng *b);
```

Comment:
Please update the docs - API docs, upgrade instructions, and CHANGELOG.

Reply:
Done, thanks. When upgrading the docs I assumed that JS and Java would use `latLngDistance` and Python would use `latlng_distance`.